packages/react/src/components/TableOfContents/README.md

```diff
@@ -59,11 +72,12 @@ DDS_TOC=true
 
 ## Props
 
-| Name        | Required | Data Type | Default Value | Description                                                                      |
-| ----------- | -------- | --------- | ------------- | -------------------------------------------------------------------------------- |
-| `children`  | NO       | Array     | null          | Content to display next to the side nav.                                         |
-| `menuItems` | NO       | Array     | null          | Array of menu item objects to render within the side nav. See `menuItems` below. |
-| `menuLabel` | NO       | String    | `Jump to`     | Placeholder value for menu label                                                 |
+| Name        | Required | Data Type | Default Value                                | Description                                                                      |
+| ----------- | -------- | --------- | -------------------------------------------- | -------------------------------------------------------------------------------- |
+| `children`  | NO       | Array     | null                                         | Content to display next to the side nav.                                         |
+| `menuItems` | NO       | Array     | null                                         | Array of menu item objects to render within the side nav. See `menuItems` below. |
+| `menuLabel` | NO       | String    | `Jump to`                                    | Placeholder value for menu label                                                 |
+| `theme`     | NO       | String    | `` | Defines the color theme for the pattern |
```

Comment:
Can you add "See `Theme` below." in the description?

Reply:
@annawen1, I've updated the readme with this, thanks!